### PR TITLE
UIEH-1075 Jest+RTL > Test SearchFilters

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Jest+RTL > Test SearchFilters

## Issues
[UIEH-1075](https://issues.folio.org/browse/UIEH-1075)

### CODE REVIEW

File: src/components/search-form/search-filters.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        label: 'Sort-label',
+        name: 'sort',
+        defaultValue: 'relevance',
+        options: [{
+          label: 'Relevance-label',
+          value: 'relevance',
+        }, {
+          label: 'Package-label',
+          value: 'package',
+        }]
```
Comment (Vladyslav-Velytskyi) [inline src/components/search-form/search-filters.test.js:29]:
```suggestion
        }],
```

File: src/components/search-form/search-filters.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import {
+  render,
+  cleanup,
+  fireEvent,
+} from '@testing-library/react';
+
+import Harness from '../../../test/jest/helpers/harness';
+import SearchFilters from './search-filters';
+
+
```
Comment (Vladyslav-Velytskyi) [inline src/components/search-form/search-filters.test.js:10]: please, remove this empty string

File: src/components/search-form/search-filters.js [hunk 3]
```diff
@@ -9,14 +9,36 @@ import {
...
+    name: PropTypes.string.isRequired,
+    options: PropTypes.arrayOf(PropTypes.shape({
+      label: PropTypes.node.isRequired,
+      value: PropTypes.string.isRequired
+    })).isRequired
+  })).isRequired,
+  closedByDefault: PropTypes.bool,
+  disabled: PropTypes.bool.isRequired,
+  onUpdate: PropTypes.func.isRequired,
+  searchType: PropTypes.string.isRequired
```
Comment (Vladyslav-Velytskyi) [inline src/components/search-form/search-filters.js:26]:
```suggestion
      label: PropTypes.node.isRequired,
      value: PropTypes.string.isRequired,
    })).isRequired,
  })).isRequired,
  closedByDefault: PropTypes.bool,
  disabled: PropTypes.bool.isRequired,
  onUpdate: PropTypes.func.isRequired,
  searchType: PropTypes.string.isRequired,
```

File: src/components/search-form/search-filters.js [hunk 4]
```diff
@@ -9,14 +9,36 @@ import {
 
 import styles from './search-form.css';
 
-export default function SearchFilters({
+const propTypes = {
+  activeFilters: PropTypes.object,
```
Comment (Vladyslav-Velytskyi) [inline src/components/search-form/search-filters.js:13]: Could you describe this object?

Comment (BogdanDenis) [reply src/components/search-form/search-filters.js]: This object is dynamic - keys are filter names and values are filter values. I've left a comment describing the object

**Review by Vladyslav-Velytskyi — APPROVED** [review]

**Review by mariiaschv — APPROVED** [review]

**Pull request merged**